docs/Using Randomness/chainlink-vrf-api-reference.md

```diff
@@ -73,10 +73,10 @@ ___
 
 # Reference
 
-## Choosing A Seed
+## Maximizing security
 
-Since the ultimate input to the VRF is mixed with the block hash of the block in which the request is made, user-provided seeds have no impact on its economic security properties. They are only included for API compatibility with previous versions of this contract.
+Chainlink VRF provides powerful security guarantees and is easy to integrate. However, smart contract security is a nuanced topic.
 
-## Maximizing security
+## Seed Depreciation
 
-Chainlink VRF provides powerful security guarantees and is easy to integrate. However, smart contract security is a nuanced topic.
\ No newline at end of file
+Since the ultimate input to the VRF is mixed with the block hash of the block in which the request is made, user-provided seeds have no impact on its economic security properties. They are only included for API compatibility with previous versions of this contract, hence it's depreciation. 
```

Comment:
I don't believe this is the case anymore?

The seed is not included for API compatibility anymore by the looks of things.

Reply:
> hence it's depreciation.

Reply:
The seed is not included in the API at all.